src/BuildScriptGenerator/DefaultBuildScriptGenerator.cs

```diff
@@ -255,13 +271,14 @@ private static string GetBenvArgs(IDictionary<string, string> benvArgsMap)
                 var snippet = platform.GenerateBashBuildScriptSnippet(context);
                 if (snippet != null)
                 {
-                    _logger.LogDebug("Script generator {scriptGenType} was used", platform.GetType());
+                    _logger.LogDebug("Platform {platformType} was used", platform.GetType());
                     snippets.Add(snippet);
                     platform.SetRequiredTools(context.SourceRepo, targetVersion, toolsToVersion);
                 }
                 else
                 {
-                    _logger.LogDebug("Script generator {scriptGenType} cannot be used", platform.GetType());
+                    _logger.LogWarning("{platformType}.GenerateBashBuildScriptSnippet() returned null",
```

Comment:
Split the arguments to a method into separate lines if you are not on a single line

Reply:
I changed the styling rule file so that this doesn't trigger a warning anymore

Reply:
Some teams (ex: ASP.NET Core) follow this pattern of either having all args on a single line or split individually...we could all agree on one approach and stick to it...